README.md

```diff
@@ -57,6 +60,7 @@ jobs:
         id: ai-review
         uses: pupilfirst/ai-review-action@v1
         env:
+          OPEN_AI_MODEL: gpt-4-turbo-preview
```

Comment:
Do we need to default to 4-turbo? can't we use `gpt-3.5-turbo`

Reply:
It is readme, we are not defaulting to `4-turbo` I had used it as an example. Should didn't ?

Reply:
Non-issue. The default is set in `app/open_ai_client.rb`, and it's `gpt-3.5-turbo`.